e2e/Dockerfile.hello-microshift

```diff
@@ -0,0 +1,19 @@
+FROM busybox:1.36
+
+ARG USERNAME=microshift
+ARG GROUPNAME=$USERNAME
+ARG USER_UID=1001
+ARG USER_GID=$USER_UID
```

Comment:
Sanity question: have you tested the image on MicroShift? Let's be sure there are no security errors when deploying Pod with this image

Reply:
Checked in `x86_64` Miroshift, the Pod was running fine with out any errors, and was able to access it through Route.